npc/dev/test.txt

```diff
@@ -151,20 +151,25 @@ function	script	F_TestVarOfAnotherNPC	{
 }
 
 function	script	F_TestCalendarNextTime	{
-	.@hour = gettime(GETTIME_HOUR);
-	.@min = gettime(GETTIME_MINUTE);
-	
-	.@result = getcalendartime(.@hour, .@min);
-	.@expected = gettimetick(2) + 24 * 60 * 60;
-	
+	.@local_hour = gettime(GETTIME_HOUR, true);
+	.@local_min = gettime(GETTIME_MINUTE, true);
+	.@result_fromlocal = getcalendartime(.@hour, .@min, -1, -1, true);
+
+	.@utc_hour = gettime(GETTIME_HOUR, false);
+	.@utc_min = gettime(GETTIME_MINUTE, false);
+	.@result_fromutc = getcalendartime(.@hour, .@min, -1, -1, false);
```

Comment:
and here `.@hour` / `.@min` is no longer defined, should be `.@local_*` and `.@utc_*`